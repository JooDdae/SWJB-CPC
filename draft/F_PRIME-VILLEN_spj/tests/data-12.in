106154135147682333735483977574189516481
