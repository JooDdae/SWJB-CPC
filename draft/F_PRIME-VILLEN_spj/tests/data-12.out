22
178 3 13 2 3 0 15 877 2 2 4 2 725 25 4 3 4 13 24 3 0 841 